src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            if (receiverOpt == null)
+            {
+                // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+                // then there's nothing left to visit anyway.
```

Comment:
In what cases would they be `null`?  Errors or normal code that would produce this?